dart/lib/src/protocol/runtime.dart

```diff
@@ -1,9 +1,13 @@
+const runtimesType = 'runtimes';
```

Comment:
just out of curiosity, why the type is not in the class?

Reply:
✅ I moved it in the class and renamed it `Runtime.listType`